src/Containers/Calendar/Containers/CalendarTablet/CalendarTablet.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { View, ViewStyle, StyleSheet } from "react-native";
+import MonthCalendar from "../MonthCalendar";
+import WeekCalendar from "src/Containers/Calendar/Components/WeekCalendar";
+import CurrentMonth from "../../Components/CurrentMonth";
+import WeekDays from "../../Components/WeekDays";
+import getShadowStyle from "src/Themes/getShadowStyle";
+
+import withTheme, { ThemeProps, ThemeParamsType } from "src/Themes/withTheme";
+
+interface DefaultProps {
+  currentDate: Date;
+}
+
+type Styles = ReturnType<typeof getStyles>;
+
+interface Props extends ThemeProps<Styles>, DefaultProps {
+  onDateChange(date: Date): void;
+  style?: ViewStyle;
+  calendarStyle?: ViewStyle;
+  submitLabel?: React.ReactNode;
+  cancelLabel?: React.ReactNode;
+}
+
+type State = {
+  isMonthCalendarOpen: Boolean;
```

Comment:
Difficult name

`isOpen` does not seem to have any naming collisions

Reply:
Still used in `src/Containers/Calendar/Containers/Calendar/Calendar.tsx:22`

And this is the exact problem of not coallocating mobile/tablet implementation (https://kentcdodds.com/blog/colocation/)

Maybe it is better to move it (mobile and table) to one module with the mobile/tablet condition in an index file and shared implementation?